fdm-app/app/components/blocks/field-map.tsx

```
@@ -0,0 +1,58 @@
+import { Map, Source, Layer } from 'react-map-gl'
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)